examples/ingressRaw.dhall

```diff
@@ -24,30 +22,30 @@ in let mkIngress : Config -> Ingress =
   \(config : Config) ->
 
   -- Given a service, make a TLS definition with their host and certificate
-     let makeTLS = \(service : Service) ->
-    { hosts = Some (List Text) [ service.host ]
-    , secretName = Some Text "${service.name}-certificate"
-    }
+	 let makeTLS = \(service : Service) ->
+	{ hosts = Some [ service.host ]
+	, secretName = Some "${service.name}-certificate"
+	}
 
   -- Given a service, make an Ingress Rule
   in let makeRule = \(service : Service) ->
-    { host = Some Text service.host
-    , http = Some RuleVal
-      { paths = [ { backend = { serviceName = service.name
-                              , servicePort = IntOrString.Int 80
-                              }
-                  , path = None Text
-                  }
-                ]
-      }
-    }
+	{ host = Some service.host
+	, http = Some
+	  { paths = [ { backend = { serviceName = service.name
+							  , servicePort = IntOrString.Int 80
+							  }
+				  , path = None
```

Comment:
Looks like CI is failing on this one, it should be `None Text`

Reply:
Also the new intendation looks a bit weird? (if you're using emacs it might be that it's adding tabs instead of spaces, it sometimes does this to me when editing dhall files)

Reply:
Yeah, it's tabs. What should be the default indentation format?

Reply:
Spaces (it's like that in the rest of the files)

Other than that, we align things in a haskell-ish way, since we cannot run `dhall format` (as it eats up the mid-file comments)